src/components/notification-box/Notification.tsx

```diff
@@ -0,0 +1,21 @@
+import Image from "next/image";
```

Comment:
Change name of parent folder
notification-box => NotificationBox

Reply:
it is notification-box